scss/components/table.scss

```diff
@@ -65,7 +65,6 @@ $block: #{$fd-namespace}-table;
         @include fd-type("-1");
         @include fd-var-color("color", $fd-table-header-color, --fd-table-header-color);
         @include fd-var-color("background-color", $fd-table-header-background-color, --fd-table-header-background-color);
-        text-transform: uppercase;
```

Comment:
I have no objection to the removal of the transform -- that worked.

Discussion topic here ... is it desired to have the table headers be a smaller font size than the base font size in the table (and on the page)?  Typically, headings and table headers (`<th>`) are usually emphasized more, not less.  Previously, even though the header font size was smaller, the emphasis was achieved by the uppercase text.

Reply:
I see that #1292 is reference, but it does not go into where the request to remove uppercase originated from. 

If #1292 originated with Fiori3 (which it *SHOULD* have) then I bet that spec addresses the font size issue that @greg-a-smith has brought up as well.

Reply:
@saad-mo @LeoT7508 ~ Do you happen to have a quick link to the reference for tables? In our current UI Kit Sketch file release we're presenting all caps on the table headers as well. Leo, maybe this is something we can update for the next Sketch file release as well?

Reply:
The request comes from the latest design update for tables. I am sure @LeoT7508 and @jeannewalters can bring more info.

Reply:
@droshev shared the mocks. it does appear the headers are larger as well. 

That said,I think *all* components are getting some changes - table and others will receive changes.

This PR does resolve the narrow issue asked for here. 🚢